functions/go/create-setters/createsetters/create_setters.go

```diff
@@ -19,6 +19,9 @@ type CreateSetters struct {
 	// ScalarSetters holds the user provided values for simple scalar setters
 	ScalarSetters []ScalarSetter
 
+	// Replacer is used to replace the setter values
```

Comment:
Seems like you are storing the scalar setters in replacer. Explain that in the comment.

// replacer holds the scalar setters info and used to efficiently generate scalar setter comments.

This should be private variable. Replace-> replacer